hapi-utils/src/main/java/com/hedera/services/recordstreaming/RecordStreamingUtils.java

```diff
@@ -53,4 +54,13 @@ public static Pair<Integer, Optional<SignatureFile>> readSignatureFile(final Str
 			return Pair.of(-1, Optional.empty());
 		}
 	}
+
+	public static Optional<SidecarFile> readSidecarFile(final String fileLoc) {
+		try (final var fin = new FileInputStream(fileLoc)) {
+			final var recordStreamSidecarFile = SidecarFile.parseFrom(fin);
+			return Optional.ofNullable(recordStreamSidecarFile);
+		} catch (Exception e) {
+			return Optional.empty();
```

Comment:
Is it safe to hide this exception? At least for testing, I'd guess we want some logs.